### PR TITLE
Fixes #issue2 and #issue3, ensure unit tests work, remove Grunt

### PR DESCRIPTION
**Pull request by schalkneethling**
@jpetto r?

### CODE REVIEW

File: js/infobar.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-$(function() {
+(function(window) {
```
Comment (jpetto) [inline js/infobar.js]: I wouldn't recommend passing in `window` in this fashion. Either rename it to something shorter (e.g. `w`) in the IIFE, or don't pass it in at all (as `window` should always refer to the global).

**Comment by jpetto** [comment]
One small suggestion, otherwise looking good. r+wc

**Comment by schalkneethling** [comment]
Thx @jpetto - updated

**Comment by jpetto** [comment]
LGTM! :baby_chick:

**Pull request merged**